iaso/api/org_unit_change_requests/views.py

```diff
@@ -218,20 +247,150 @@ def export_to_csv(self, request):
                 change_request.org_unit.name,
                 change_request.org_unit.parent.name if change_request.org_unit.parent else None,
-                change_request.org_unit.org_unit_type.name,
+                change_request.org_unit.org_unit_type.name if change_request.org_unit.org_unit_type else None,
                 ",".join(group.name for group in change_request.org_unit.groups.all()),
                 change_request.get_status_display(),
                 datetime.strftime(change_request.created_at, "%Y-%m-%d"),
                 get_creator_name(change_request.created_by) if change_request.created_by else None,
                 datetime.strftime(change_request.updated_at, "%Y-%m-%d"),
                 get_creator_name(change_request.updated_by) if change_request.updated_by else None,
+                change_request.get_status_display(),
             ]
+
+            # Name changes
+            name_before = change_request.old_name if change_request.kind == change_request.Kind.ORG_UNIT_CHANGE else ""
+            name_after = change_request.new_name if change_request.new_name else change_request.org_unit.name
+            name_conclusion = get_conclusion(name_before, name_after)
+
+            row.extend([name_before, name_after, name_conclusion])
+
+            # Parent changes
+            parent_before = change_request.old_parent.name if change_request.old_parent else ""
+            parent_after = (
+                change_request.new_parent.name
+                if change_request.new_parent
+                else change_request.org_unit.parent.name
+                if change_request.org_unit.parent
+                else None
+            )
+
+            row.extend([parent_before, parent_after])
+
+            # Reference extensions for parents
+            for level in range(1, 4):
+                parent_before = change_request.old_parent if change_request.old_parent else None
+                parent_after = (
+                    change_request.new_parent if change_request.new_parent else change_request.org_unit.parent
+                )
+
+                ref_ext_before = get_parent_ref_ext(parent_before, level)
+                ref_ext_after = get_parent_ref_ext(parent_after, level)
+                ref_ext_conclusion = get_conclusion(ref_ext_before, ref_ext_after)
+
+                row.extend([ref_ext_before, ref_ext_after, ref_ext_conclusion])
+
+            # Opening date changes
+            opening_date_before = (
+                change_request.old_opening_date.strftime("%Y-%m-%d") if change_request.old_opening_date else ""
+            )
+            opening_date_after = (
+                change_request.new_opening_date.strftime("%Y-%m-%d")
+                if change_request.new_opening_date
+                else (
+                    change_request.org_unit.opening_date.strftime("%Y-%m-%d")
+                    if change_request.org_unit.opening_date
+                    else None
```

Comment:
I think here it should return empty string instead of None to be able to compare the before and after opening date